service/matching/matchingEngine.go

```diff
@@ -495,16 +494,6 @@ func (e *matchingEngineImpl) QueryWorkflow(ctx context.Context, queryRequest *ma
 		}
 
 		workerResponse := result.workerResponse
-		// if query was intended as consistent query check to see if worker supports consistent query
-		if queryRequest.GetQueryRequest().GetQueryConsistencyLevel() == enums.QueryConsistencyLevelStrong {
-			if err := e.versionChecker.SupportsConsistentQuery(
-				workerResponse.GetCompletedRequest().GetWorkerVersionInfo().GetImpl(),
-				workerResponse.GetCompletedRequest().GetWorkerVersionInfo().GetFeatureVersion()); err != nil {
-				// TODO: this error is swallowed and client gets "deadline exceeded" instead.
```

Comment:
I remember, I spent 3 days fighting with this check. I should remove it before we started.